app/models/forms/export.rb

```diff
@@ -218,7 +243,11 @@ def to_xls
                 q.question.hint_translations&.dig(locale.to_s))
             end
             questions.row(row_index).push(q.code, q.required.to_s,
-              conditions_to_push, constraints_to_push, choice_filter)
+              conditions_to_push, choice_filter, constraints_to_push)
+
+            constraint_msg_to_push.each do |message|
+              questions.row(row_index).push(message)
+            end
```

Comment:
rather than having a separate `each` loop here, it may be possible to combine these: 

`[].push(1, 2, *[3, 4])` is equivalent to `[].push(1, 2, 3, 4)`, with the `*` "splat" operator deconstructing the array